packages/tools/locales/de_DE.po

```diff
@@ -124,11 +126,11 @@ msgstr "Herunterladen"
 
 #: packages/app-desktop/checkForUpdates.js:189
 msgid "Skip this version"
-msgstr ""
+msgstr "Überspringe diese Version"
 
 #: packages/app-desktop/checkForUpdates.js:189
 msgid "Full changelog"
-msgstr ""
+msgstr "Kompletter Changelog"
```

Comment:
Maybe something like `Komplette Versionsinformation` might be better. But I really don't know.

Reply:
"Komplette Versionsinformation" sounds better than "kompletter Changelog".
According to wikipedia: changelog = "Änderungsprotokoll"
[Wikipedia Changelog/Änderungsprotokoll](https://de.wikipedia.org/wiki/%C3%84nderungsprotokoll)

Reply:
You can also use `Komplettes Änderungsprotokoll`. I just know that many Germans do not like English terms in translations.

However, I'm not a good person to judge since I've never used any localized SW ever.